packages/core/variables/_colors.scss

```diff
@@ -45,3 +27,25 @@ $varslingsfarger: (
 @function varslingsfarge($farge) {
     @return map.get($varslingsfarger, $farge);
 }
+
+$stottefarge-gronn: #578363;
+$stottefarge-rod: #9d3a14;
+
+// Meldingsfarger for tekst og ikoner
+$info: map.get($varslingsfarger, "info");
+$advarsel: map.get($varslingsfarger, "advarsel");
+$suksess: $stottefarge-gronn;
+$feil: $stottefarge-rod;
```

Comment:
Vil vi dette? Så vidt jeg har skjønt har det vært et ønske fra designerne om at bruk av disse fargene skal begrenses utenfor definerte komponenter i Jøkul. Det er i hvert fall grunnen til at vi _ikke_ eksponerer `$varslingsfarger` gjennom `jkl.scss` i dag.

Reply:
De gamle fargene er eksponerte i dag igjennom klasser. Ikke dermed sagt vi trenger å fortsette med det, men jeg tenkte spare en breaking change i core til noe litt mer substansielt 😅 Det sagt: jeg funderte på å heller sette `info` og `advarsel` til standard tekstfarge som en "deprecate". Kontrasten til de nye fargene er jo ikke spesielt god på lys bakgrunn.

Skjemakomponentene bruker i hvertfall `$feil` og `$feil--darkbg` for feilmeldinger, så de må fortsatt kunne nås. Tanken var også at `stottefarge-gronn` kunne brukes til checkmarks f. eks. i en dekningstabell.

Reply:
Hvis skjemakomponentene bruker `$feil` og `$feil--darkbg` synes jeg heller vi skal refaktorere dem til å bruke `map.get($varslingsfarger, "feil--alt")` og `map.get($varslingsfarger, "feil")`.

Jeg ser jo greia med å ikke ville ha breaking changes i hver release, men synes kanskje også det blir feil å eksponere de nye fargene dersom det ikke er meningen? Da synes jeg det er bedre å fortsette å eksponere de gamle fargene (og merke dem deprecated) 🤷‍♂️

Reply:
BTW, ut fra @Slaine2 sin kommentar i standup i dag virker det som de to støttefargene er ment å fylle funksjonen som `$varslingsfarger.feil--alt` og `$varslingsfarger.suksess--alt` er ment å dekke. kanskje de kan konsolideres på en måte? De skal vel heller ikke brukes utover eksisterende komponenter, så vidt jeg har skjønt?

Reply:
I morges diskuterte vi problemet igjen på VD-forumet og tok et skritt tilbake. Vi har tenkt gjennom de ulike applikasjonene på nytt og har åpnet vinduet igjen. Det kan godt hende at vi kun bruker støttefargene bare til checkmarks og løser hjelpeteksten annerledes. men siden vi har mange forskjellige meninger, ønsker vi å teste dette først.